src/overlays/actors/ovl_Obj_Tokei_Step/z_obj_tokei_step.c

```diff
@@ -269,7 +268,7 @@ void ObjTokeiStep_Update(Actor* thisx, GlobalContext* globalCtx) {
 void ObjTokeiStep_Draw(Actor* thisx, GlobalContext* globalCtx) {
     ObjTokeiStep* this = THIS;
 
-    func_800BDFC0(globalCtx, D_06000088);
+    func_800BDFC0(globalCtx, gClocktowerPanelDL);
 }
 
 void ObjTokeiStep_DrawOpen(Actor* thisx, GlobalContext* globalCtx) {
```

Comment:
`s32` instead of `int` just below here. I didn't see any others, but do make sure all the integer types are the ultratypes.